core/domain/entities/shortcodes/EspressoEventAttendees.php

```diff
@@ -100,21 +104,21 @@ public function initializeShortcode()
      * @param array $attributes
      * @return string
      * @throws EE_Error
-     * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
-     * @throws \EventEspresso\core\exceptions\InvalidInterfaceException
-     * @throws \InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws InvalidArgumentException
+     * @throws DomainException
      */
     public function processShortcode($attributes = array())
     {
         // grab attributes and merge with defaults
         $attributes = $this->getAttributes((array) $attributes);
-        $archive = is_archive();
+        $attributes['limit'] = (int) $attributes['limit'];
         $display_on_archives = filter_var($attributes['display_on_archives'], FILTER_VALIDATE_BOOLEAN);
         // don't display on archives unless 'display_on_archives' is true
-        if ($archive && ! $display_on_archives) {
+        if ($attributes['limit'] === 0 || (! $display_on_archives && is_archive())) {
```

Comment:
so is the change here mostly that the a limit of 0 is another way of "turning off" a render (shortcode or block)?

Reply:
ya basically... if someone sets the limit to zero, then they obviously don't want any results posted